### PR TITLE
Initialize the ScholarX-backend

### PR DESCRIPTION
**Pull request by janithlahirukariyawasam**
The goal of this PR is to initialize the ScholarX-backend. 
Issue number  #2

### CODE REVIEW

**Comment by janithlahirukariyawasam** [comment]
@anjula-sack can review this?

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Looks good! Can you configure the following as well,

Jest for testing
Eslint for lining
Prettier for code formatting

**Comment by janithlahirukariyawasam** [comment]
Can you check again @anjula-sack

File: package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "scholarxbackend",
```
Comment (Piumal1999) [inline package.json:2]: Shall we change this to scholarx-backend? I think it looks better that way.
```suggestion
  "name": "scholarx-backend",
```

Comment (Piumal1999) [reply package.json]: Run the npm install command once, so it will update the name on the package-lock.json as well.

Comment (janithlahirukariyawasam) [reply package.json]: Done

**Review by Piumal1999 — APPROVED** [review]
Looks good to me

**Review by anjula-sack — APPROVED** [review]
Good job! @janithlahirukariyawasam

**Pull request merged**